src/sora-v3.md

```diff
@@ -0,0 +1,116 @@
+
+## SORA V3 Tokenomics
+
+The SORA V3 tokenomics were defined over two iterations, the first, called for a suspension of TBCD minting while production was funded through the now defunct VXOR token. The second and current iteration uses the KUSD Dollar pegged synthetic asset to fund production, while implementing buyback and burn of this asset within network transactions. Here is a detailed overview of both iterations.
+
+### First Iteration
+
+![](/.gitbook/assets/sora-v3-2.png)
+
+Economies are like organisms; they are functioning, living entities and must constantly grow and evolve to stay alive. This means that money, the lifeblood of an economy, needs to provide sufficient nourishment to producers while also not increasing beyond the size needed to sustain health.
+
+In SORA v2, the SORA (XOR) token (the main token on the network, used to pay gas fees) had an elastic supply managed by a token bonding curve. The theory behind this was sound — to manage the token supply to meet the economy's needs. However, without sufficient reserves, the SORA v2 economy experienced liquidity shortages.
+
+At the same time, countries worldwide have been debasing their currencies by expanding their monetary bases for non-productive uses. While not directly related to SORA, this is an important narrative for billions of people, and (perhaps intentionally), many people became scared of any expansion in currency supplies, even for productive uses. This meta-narrative has important implications for SORA because we have to work within the confines of the grand narrative created globally, as we are, as of yet, too small to direct it ourselves.
+
+Because of the state of the world and the needs that people all over the world have, SORA v3 must be able to provide adequate funding for producers to target high GDP growth in the SORA economy while also limiting the increase of token supply expansion, given the meta-narrative that has caused people to be fearful of “muh inflation.”
+
+Builders in the SORA ecosystem under the v3 tokenomics will still get paid for their work, but not in XOR. Instead, they will be paid in tokens (probably changing monthly or quarterly) that will be accepted as collateral to create Kensetsu XOR, a stablecoin pegged to the value of XOR. Kensetsu XOR will have the liquidity builders need because it will be usable as if it were real XOR in Chameleon pools (initially XOR-ETH only and configurable by on-chain governance). LPs of Chameleon pools will farm KARMA tokens to encourage Chameleon pool liquidity, as explained in a [recent article by Satoshi Shibarimoto](https://medium.com/@shibarimoto/earn-karma-with-a-sora-chameleon-01b25c12fd49).
```

Comment:
I think this paragraph is outdated. 
The concrete tokenomics for v3 are still under investigation as I understand.